### DISCUSSION TITLE
Integrate my mod SeeContracts?

### DISCUSSION THREAD

**Discussion by ghost** [Royal-Modding/FS19_BetterContracts#8] [Ideas]
Hi @TyKonKet and Royal Modding Team,

I have just finished my FS19 script mod SeeContracts that enhances the ingame contracts page with additional info. I tested compatibility with FS19_BetterContracts. Before I submit it to modhub for Giants QA, I would like to discuss with you:

- do you see a chance to integrate my script to your mod?
- since I cannot test multiplayer, I'd like to learn - what sort of test environment have you set up?
- is it possible to join your team - what would be your requirements?

I have just uploaded my source and the initial release here in a private repository, and will add you as collaborator. So that you can have a look at it. Please take your time, I am in no kind of a hurry ;)

Kind Regards, Mmtrx

**Comment by Lactic68** [comment]
So where is going this collaboration proposal ? Your two mods seems to be a big step in contract management in game.
Any news about your work in progress ?

**Comment by ghost** [comment]
Well, I haven't used github for a long time. Just trying to remember things.. I just have forked an own branch of BetterContracts, and will try to include my additional gui displays there. But I can only test it on single player. So - when I am finished with the integration, I will need your support to add / test the MP parts (events, readStream, writeStream). Here is a screen of my w.i.p. ![screen of my wip](https://user-images.githubusercontent.com/51960039/115518826-e8356f00-a288-11eb-837d-54094cf8bbf4.png)

**Reply by TyKonKet** [reply]
Hey @mhausberg do you have discord?
I would like to talk with you privately
